app/jinja_filters.py

```diff
@@ -256,9 +256,25 @@ def should_wrap_with_fieldset_processor() -> dict[str, Callable]:
 @blueprint.app_template_filter()
 def get_width_for_number(answer: AnswerType) -> Optional[int]:
     allowable_widths = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 40, 50]
+    min_value = None
+    max_value = None
 
-    min_value = answer.get("minimum", {}).get("value", 0)
-    max_value = answer.get("maximum", {}).get("value", MAX_NUMBER)
+    schema = g.get("schema")
+
+    if answer.get("minimum", {}) and isinstance(answer["minimum"]["value"], dict):
```

Comment:
@petechd I was thinking about this from the conversation in tech session - what if it was something on these lines?

```
min_value = answer.get("minimum", {}).get("value")
while isinstance(min_value, dict):
    source = schema.get_answers_by_answer_id(min_value["identifier"])
    min_value = source[0].get("minimum", {}).get("value")
```

and same for max_value - it doesn't account for finding multiple variant answers, but as a general idea I think it'd be possible to do this without the `min_and_max_map` dictionary
(also that check might need to be more careful than just seeing if its a dictionary, but just an initial idea)

Reply:
I am all for this idea, as it seems better, but I am not sure if we can do something that is different from the initial ask on the card.

Reply:
ahh I hadn't checked the comment about making it at schema load time, perhaps suggested that way for performance? in which case yeah this might not be viable